### PR TITLE
LPD-40537 Fix symlinks

### PR DESCRIPTION
**Pull request by petershin**
@4lejandrito

### CODE REVIEW

**Comment by petershin** [comment]
ci:test:sf

**Comment by petershin** [comment]
ci:test:relevant

**Comment by petershin** [comment]
ci:test:gradle-plugins

File: modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/RESTBuilder.java [hunk 1]
```diff
@@ -1983,18 +1983,31 @@ private String _getNodePrefix() throws Exception {
...
 		}
 
 		return String.valueOf(
 			Paths.get(
-				absolutePathString.substring(0, index), "liferay-portal",
-				"build", "node", "bin", "npm"));
+				portalDir.getAbsolutePath(), "build", "node", "bin", "npm"));
+	}
+
+	private File _getPortalDir(File dir) {
```
[the remark this answers is not in the document]
Comment (petershin) [reply modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/RESTBuilder.java:1997]: @4lejandrito - thank you for testing this!

I'll follow up w Drew on how to add support in workspaces. Is there an existing ticket for this?

**Comment by petershin** [comment]
ci:forward

File: modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/RESTBuilder.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (4lejandrito) [reply modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/RESTBuilder.java:1997]: You can use https://liferay.atlassian.net/browse/LPD-41343.

**Comment by 4lejandrito** [comment]
Hey @petershin, I tested this and it works. If I understand correctly what this does is mark the output of `downloadNode` as never up to date, essentially disabling the cache. Wouldn't it be easier just to remove the [@CacheableTask](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/task/DownloadNodeTask.java#L39) annotation? 

I that case we losing the cache benefits for downloading node... is it worth it? It's a big download and I think using the cache is important for this task.

**Pull request closed without merging**